components

Wikitelmex opened the pull request:
- delete boilerplate elements
- add calculator component
- add lazy loading to calculator component
- add styling to calculator component
- call calculator component from APP component

wikitelmex1 committed:
components

- delete boilerplate elements
- add calculator component
- add lazy loading to calculator component
- add styling to calculator component
- call calculator component from APP component

wikitelmex1 committed:
linters

- fix style linters

eliasecasta commented on src/components/Calculator/Calculator.js:
```diff
@@ -0,0 +1,35 @@
+import React, { useState } from 'react';
+import './Calculator.css';
+
+const Calculator = () => {
```
- [x] This calculator component should be class-based, please adapt it accordingly to comply with the project requirements.

Wikitelmex replied:
fixed

eliasecasta commented on src/App.js:
```diff
@@ -1,5 +1,6 @@
 import './App.css';
+import Calculator from './components/Calculator/Calculator.lazy';
 
 function App() {
```
- [x]  This app component should be class-based, please adapt it accordingly to comply with the project requirements.

Wikitelmex replied:
fixed, ant tyvm for the review 😄

eliasecasta reviewed (changes requested):
# Status:  Changes Required ♻️
_Check the comments under the review_
<hr>

Hello @Wikitelmex, I am Elias, your reviewer for this PR

Well done so far with your project!

The calc styling and the animation look awesome, great job so far! 🥇 

I will mention to you a small summary of things that need to be changed in order to keep moving:

- [x] Please remove all the boilerplate files created by the create react app command, this includes your favicon.ico in your public folder and the index.css file which is not being used by you and does not affect the program by being removed so we can comply with the project requirements.
- [x] Please check the comments below this PR for more requirements.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please correct those issues and send another review request, good luck, and well done so far! 

Code review by Elias Castañeda

_(Feel free to leave any questions or comments in the PR thread if something is not 100% clear)_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/restricted?return_to=https%3A%2F%2Fmicroverse.zendesk.com%2Fhc%2Fen-us%2Farticles%2F1500004088561) you have a limited number of code reviews per project (you can check the exact number in your Dashboard). If you think that the code review was not fair, you can [request a second opinion using this form](https://airtable.com/shrQAqnBwek5a0O0s).

wikitelmex1 committed:
Changes required

- delete boilerplate elements (index.css is currently used)
- fix on lazy loading to match class based component
- change calculator function to calculator class

DeliceLydia reviewed (approved):
# Congratulations🎉🎉

## Hi @Wikitelmex, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
